docusaurus/docs/dev-docs/configurations/environment.md

```diff
@@ -24,6 +24,9 @@ Strapi provides the following environment variables:
 | `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` <br/><br/>_Optional_ | Initialization locale for the application, if the [Internationalization (i18n) plugin](/dev-docs/plugins/i18n) is installed and enabled on Content-Types (see [Configuration of i18n in production environments](/dev-docs/plugins/i18n#configuration-of-the-default-locale)) | `String`  | `'en'`          |
 | `FAST_REFRESH`                                             | Use [react-refresh](https://github.com/pmmmwh/react-refresh-webpack-plugin) to enable "Fast Refresh" for near-instant feedback while developing the Strapi admin panel.                                                                                                       | `boolean` | `true`          |
 
+:::tip
+Prefixing an environment variable name with `STRAPI_ADMIN` exposes the variable to the admin front end.
```

Comment:
might be worth distinguishing that it exposes it on `process.env` not on `window.strapi` as far as i can tell from the code, you would need to manually add it to `window.strapi`.

Reply:
Thanks, Josh. [Fixed it](https://github.com/strapi/documentation/pull/1631/commits/bc84d8cc0c4d5822d27119bef05f795bd493cec8). WDYT?